SConscript

```diff
@@ -0,0 +1,13 @@
+Import('RTT_ROOT')
+Import('rtconfig')
+from building import *
+
+objs = []
+cwd     = GetCurrentDir()
+
+src     = Glob('hydrogen.c')
```

Comment:
For a single file, globbing is probably not required :)

Reply:
Yes, I'll update this.